server/server.go

```diff
@@ -0,0 +1,173 @@
+package server
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"net"
+
+	ipfslite "github.com/hsanjuan/ipfs-lite"
+	"github.com/ipfs/go-cid"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	pb "textile.io/grpc-ipfs-lite/ipfs-lite"
+)
+
+type ipfsLiteServer struct {
+	pb.UnimplementedIpfsLiteServer
+
+	// should this be a file var or here?
+	peer *ipfslite.Peer
```

Comment:
By `file var` you mean a package-scope variable?
Considering that is received in `StartServer` seems to be more bundled with this struct. Theoretically may make sense to call `StarServer` multiple times (in real-word, or in tests) and a file var may be handcuffing.
TL;DR: seems good for me as it is

Reply:
Cool, thanks!